lib/tty.js

```diff
@@ -40,7 +40,8 @@ const {
 let readline;
 
 function isatty(fd) {
-  return NumberIsInteger(fd) && fd >= 0 && isTTY(fd);
+  return NumberIsInteger(fd) && fd >= 0 && fd <= 2147483647 &&
+         isTTY(fd);
```

Comment:
```suggestion
  return isUint32(fd) && isTTY(fd);
```

The function can be found in `internal/validators`.

Reply:
I just noticed that it's going to include 2 ** 31 and that should not be the case.